go.mod

```diff
@@ -100,6 +100,7 @@ require (
 	github.com/zclconf/go-cty v1.3.1
 	go.uber.org/zap v1.10.0
 	golang.org/x/crypto v0.0.0-20200220183623-bac4c82f6975
+	golang.org/x/net v0.0.0-20200202094626-16171245cfb2
```

Comment:
i looked this up and this is already a transitive dependency so it is already present in go.sum and vendor. Thats why making it a direct dependency only results in a change to go.mod 👍